src/App.js

```diff
@@ -19,10 +19,10 @@ function App() {
       <Routes>
         {" "}
         <Route path="/" element={<Home />} />
-        <Route path="/countryedit" element={<CountryEdit />} />
-        <Route path="/countryindex" element={<CountryIndex />} />
-        <Route path="/countrynew" element={<CountryNew />} />
-        <Route path="/countryshow" element={<CountryShow />} />
+        <Route path="/countryedit" element={<CountryEdit countries={countries} />} />
```

Comment:
Not required for the read functionality...but since it is here...please look at this path and the props you will need to available to you to perform the update functionality.